.github/workflows/pytest.yml

```diff
@@ -129,7 +129,7 @@ jobs:
         # cmake is needed for certain CI environments where it is not preinstalled.
         # libiconv and ffmpeg are needed by WhisperX.
         if: ${{ matrix.uv-options == '' }}
-        run: ./scripts/retry.sh 3 30 conda install -c conda-forge 'cmake>=3.22' libiconv ffmpeg
+        run: ./scripts/retry.sh 3 30 conda install -c conda-forge 'cmake>=3.22' libiconv 'ffmpeg>=6,<7'
```

Comment:
Also make this change in `nightly.yml` or the nightly tests will break.

I'm thinking we might want to hard-code this to a specific version (say `6.1.2`) both here and in the Makefile, to ensure parity between dev/CI (so we don't end up in another situation where something works in dev but then fails in CI)

Reply:
Done.